docs/redoc/master/openapi.json

```diff
@@ -7358,12 +7402,14 @@
             "description": "Point version",
             "type": "integer",
             "format": "uint64",
-            "minimum": 0
+            "minimum": 0,
+            "example": 1
           },
           "score": {
             "description": "Points vector distance to the query vector",
             "type": "number",
-            "format": "float"
+            "format": "float",
+            "example": 1.25
```

Comment:
If we are using cosine, it should <1.0. I think it would be more intuitive overall